Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/main.kt

```diff
@@ -270,31 +278,48 @@ private fun processCLib(args: Array<String>, additionalArgs: Map<String, Any> =
     manifestAddend?.parentFile?.mkdirs()
     manifestAddend?.let { def.manifestAddendProperties.storeProperties(it) }
 
-    if (flavor == KotlinPlatform.JVM) {
-
-        val outOFile = tempFiles.create(libName,".o")
-
-        val compilerCmd = arrayOf(compiler, *stubIrContext.libraryForCStubs.compilerArgs.toTypedArray(),
-                "-c", outCFile.absolutePath, "-o", outOFile.absolutePath)
-
-        runCmd(compilerCmd, verbose)
-
-        val outLib = File(nativeLibsDir, System.mapLibraryName(libName))
-
-        val linkerCmd = arrayOf(linker,
-                outOFile.absolutePath, "-shared", "-o", outLib.absolutePath,
-                *linkerOpts)
+    val compilerArgs = stubIrContext.libraryForCStubs.compilerArgs.toTypedArray()
+    val nativeOutputPath: String = when (flavor) {
+        KotlinPlatform.JVM -> {
+            val outOFile = tempFiles.create(libName,".o")
+            val compilerCmd = arrayOf(compiler, *compilerArgs,
+                    "-c", outCFile.absolutePath, "-o", outOFile.absolutePath)
+            runCmd(compilerCmd, verbose)
+
+            val outLib = File(nativeLibsDir, System.mapLibraryName(libName))
+            val linkerCmd = arrayOf(linker,
+                    outOFile.absolutePath, "-shared", "-o", outLib.absolutePath,
+                    *linkerOpts)
+            runCmd(linkerCmd, verbose)
+            outOFile.absolutePath
```

Comment:
Shouldn't it be more like `outLib.absolutePath`?

Reply:
Yeah, it’s a typo.